aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ConnectEvent.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.amazonaws.services.lambda.runtime.events;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * Class to represent an Amazon Connect contact flow event.
+ *
+ * @see <a href="https://docs.aws.amazon.com/connect/latest/adminguide/connect-lambda-functions.html>Connect Lambda Functions</a>
+ *
+ * @author msailes <msailes@amazon.co.uk>
+ */
+@NoArgsConstructor
+@Data
+public class ConnectEvent implements Serializable, Cloneable {
+
+    private Details details;
+    private String name;
+
+    @NoArgsConstructor
+    @Data
+    public static class Details implements Serializable, Cloneable {
+        private ContactData contactData;
+        private Parameters parameters;
```

Comment:
Looking at the [url above](https://docs.aws.amazon.com/connect/latest/adminguide/connect-lambda-functions.html) regarding the `parameters` field:
- These are parameters specific to this call that were defined when you created the Lambda function.
In fact, looking at this [other example](https://aws.amazon.com/blogs/contact-center/creating-data-driven-ivrs-in-amazon-connect-using-a-rest-api/) it seems that the `parameters` field should be a `Map<String, Object>`. What do you think?

Reply:
fixed